api/middleware/index.js

```diff
@@ -18,7 +18,7 @@ module.exports = function(req, res, next) {
     const bearerToken = bearer[1] // <token>
 
     // Verify token
-    jwt.verify(bearerToken, 'secret', function(err, decoded) {
+    jwt.verify(bearerToken, 'FsT8VApMlKjmCgUPKIrv', function(err, decoded) {
```

Comment:
Same here, environment variable instead of hardcoded

Reply:
Whoops, meant to do that before creating the PR.